README.md

```diff
@@ -194,6 +194,12 @@ final roomOptions = RoomOptions(
 
 final room = Room();
 
+// if you are using livekit cloud, you can use `prepareConnection`
```

Comment:
I think prepareConnection can be used regardless of livekit cloud ? 
Only the region pinning support needs livekit cloud, I think?

Reply:
Ah, I see.